medical-portal/src/API/RSBC.DMF.MedicalPortal.API/appsettings.Development.json

```diff
@@ -8,5 +8,6 @@
         "Microsoft.Hosting.Lifetime": "Information"
       }
     }
-  }
+  },
+  "cms": "https://rsbc-dfp-cms-adapter-dev.apps.silver.devops.gov.bc.ca/"
```

Comment:
We've been keeping this type of configuration out of git, can you remove this please?